etc/kayobe/stackhpc-lvm.yml

```diff
@@ -0,0 +1,117 @@
+---
```

Comment:
Might need to move all of this to group_vars, since it's not possible to override these variables on a per-group basis. e.g. `stackhpc_lvm_group_rootvg_disks`, and `stackhpc_lvm_group_rootvg_lvs_extra`.